src/native_snd.rs

```diff
@@ -81,6 +96,9 @@ impl<T: Send + 'static> SoundDriver<T> {
             device.name().unwrap_or("no device name".into()),
             &output_format
         );
+        if output_format.sample_rate.0 != 44100 {
```

Comment:
What is the significance of having a sample rate of 44100? I have multiple wav files with 705kbps or even 2304kbps

Reply:
Bitrate 705kbps is exactly 44100 samples per second (44100 samples per second * 16 bits per second).

AFAIK web audio uses sample rate 44100 as a standart. Having a library multiplatform we should try our best to achieve it on Linus, Windows, Mac OS, and any other platform as well.

Knowing so, it's matter to have output sample rate 44100 and input sound sources as some multiple of 11025 so we can mix it easily. When you have output format with samle rate 48000 and sound source with sample rate 44100 (or reverse), you have to resample one to another which is nasty and in a naive implementation will result in significant sound quality loss. Big commercal digital audio workstation software is capable in doing such tasks, but it's inapropriate for a small sound library for games.

As you may guess 2304kbps has sample rate 144000 wich is not a multiple of 11025, so it should be avoided. It's also has very big size in memory, which is entirely unneeded for usual game development tasks